### ISSUE TITLE
Developing on Windows But Deploying to iOS

### ISSUE THREAD

**Issue by spentak** [heroiclabs/nakama-unreal#16]
When I build my Unreal project using the iOS binary, we get build errors.  When I use the Win64 binary it works fine.

To deploy to iOS am I suppose to pull down the code on a Mac, remove the Win64 binary and then rebuild with the iOS binary?

What is the suggested workflow here?

**Comment by Dimon4eg** [comment]
As far I know, you cannot build for iOS on Windows.
Yes, you should build for Windows on Windows PC and for iOS/Mac on Mac OSX.

**Comment by spentak** [comment]
I guess what you are saying is there needs to build conditional building, if *developing* have it somehow use the Windows Nakama plugin, and if building to device use the iOS Nakama plugin

**Comment by Dimon4eg** [comment]
Yes, also you can merge Nakama windows plug-in with iOS/Mac one. Only difference in libs, so it's easy.
